src/actions/guides/frontend/asset-handling.cr

```diff
@@ -125,6 +125,22 @@ class Guides::Frontend::AssetHandling < GuideAction
 
     Make sure to use the `asset` macro to get fingerprinted assets.
 
+    ## Asset host
+
+    Once your app is in production, you may want to serve up your assets through a [CDN](https://en.wikipedia.org/wiki/Content_delivery_network). To specify a differnt host, you'll use the `asset_host` option in `config/server.cr`.
+
+    ```crystal
+    # config/server.cr
+    Lucky::Server.configure do |settings|
+      if Lucky::Env.production?
+        settings.asset_host = "https://mycdnhost.com"
+      else
+        # Serve up assets locally in development
+        settings.asset_host = ""
+      end
+    end
```

Comment:
This looks good. I also think we should add a default in Lucky CLI that sets the `asset_host = ""` and includes a link back to here and some brief instructions on how to use it

Reply:
Not a blocker though. I'll add an issue

Reply:
https://github.com/luckyframework/lucky_cli/issues/347